docs/virtual_machines/disks_and_volumes.md

```diff
@@ -906,17 +906,17 @@ additional context switching is incurred for each thread.
 
 Disks with `dedicatedIOThread` set to `true` will not use the shared
 thread, but will instead be allocated an exclusive thread. This is
-generally useful if a specific Disk is expected to have heavy I/O
-traffic, e.g. a database spindle.
+generally useful if a specific Disk is expected to have heavy I/O traffic
+ie. a database spindle.
```

Comment:
e.g. is technically more correct here. a database spindle isn't the only case, just an example of one.

Reply:
Reverting the txt passes non-ascii check.  I'll just do that and keep the diff minimal